opentelemetry-api/src/opentelemetry/util/_providers.py

```diff
@@ -19,6 +19,7 @@
 from pkg_resources import iter_entry_points
 
 if TYPE_CHECKING:
+    from opentelemetry._metrics import MeterProvider
```

Comment:
Curious why do we need this import here? :thinking:

Reply:
Don't we need to add `MeterProvider` to the `Union` below?

Reply:
That didn't work since it has to be either one of them not the union. I wanted to get help with this and another issue shown in mypy test run.